public/app/core/components/ThemeSelector/ThemeSelectorDrawer.tsx

```diff
@@ -0,0 +1,121 @@
+import { css } from '@emotion/css';
+
+import { GrafanaTheme2, ThemeContext, ThemeRegistryItem } from '@grafana/data';
+import { Drawer, RadioButtonDot, TextLink, useStyles2, useTheme2 } from '@grafana/ui';
+import { t, Trans } from 'app/core/internationalization';
+import { changeTheme } from 'app/core/services/theme';
+
+import { ThemePreview } from '../Theme/ThemePreview';
+
+import { getSelectableThemes } from './getSelectableThemes';
+
+interface Props {
+  onClose: () => void;
+}
+
+export function ThemeSelectorDrawer({ onClose }: Props) {
+  const styles = useStyles2(getStyles);
+  const themes = getSelectableThemes();
+  const currentTheme = useTheme2();
+
+  const onChange = (theme: ThemeRegistryItem) => {
+    changeTheme(theme.id, true);
```

Comment:
would it be better to also set the new value in the backend? i think it's confusing that this only happens at runtime right now 🤔

Reply:
@ashharrison90 what do you mean?? changeTheme updates the backend stored theme preferences

Reply:
@ashharrison90 but there was a bug in changeTheme where it did not save using the passed theme id but just use the dark / light mode, fixed that and it saves now